src/commands/definition.ts

```diff
@@ -0,0 +1,39 @@
+import * as Discord from 'discord.js';
+
+const urban = require('relevant-urban');
```

Comment:
Please try to refrain from using '`require`' and use `import`. It's quite well explained in [this post](https://stackoverflow.com/questions/46677752/the-difference-between-requirex-and-import-x), why latter is preferred!